Nidhi_Gupta/progress.md

```diff
@@ -3,78 +3,79 @@
 **Project: Comments for Responses**
 
 Overall Direction:
-- 
-- 
-- 
+- Ability for respondents to add comments for their own responses in following question types.
+    * Multiple-choice (single answer) question
+    * Multiple-choice (multiple answers) question
+    * Numerical-scale question
+    * Distribute points (among options) question
+    * Distribute points (among recipients) question
+    * Rubric question
+    * Rank (Options) question
+    * Rank (Recipients) question
+
+- Ability for instructors to add comments from more places.
+	* Giver > Question > Recipient
+	* Recipient > Question > Giver
+
+Bonus features (To do after adding basic comment feature for all question types)
+- Allow instructors to disable comments.
+- Ability to know if the comment was read.
+- Allow instructors to require comments.
 
 ---
 
-Week 1 Plan:
-- [ ] 
-- [ ] 
-- [ ] 
+Week 1 Plan: 
+- [ ] Implement the backend part of the ability for respondents to add comments for MCQ questions
```

Comment:
Do your MCQ questions here referring to both the single answer and multiple answers? If yes, it may be easier for you to focus on one first and then the rest of the question types can just follow.

Reply:
I was planning to include both single and multiple answers. 
Okay I'll first do MCQ, then MSQ. Updating the plan accordingly.